.github/workflows/build.yml

```diff
@@ -142,7 +144,9 @@ jobs:
 
   build-dockers:
     runs-on: ubuntu-latest
-    needs: [build, build-arm]
+    needs: [build]
```

Comment:
If I'm reading right this will allow the job to start before `build-arm` completes when running on `main`? But that won't work

Reply:
Yes, commented that here too: https://github.com/EspressoSystems/espresso-sequencer/pull/2231#discussion_r1820960902